### ISSUE TITLE
Loading saved games

### ISSUE THREAD

**Issue by DirvingWheel** [ValveSoftware/portal2#185]
When you start playing, you can't choose the save file you want, because Pause Menu shows "Load last save" instead of "Load Game".

**Comment by Tele42** [comment]
This appears to be the intended behavior, Load Game is in the main menu.

**Comment by DirvingWheel** [comment]
Nope, it wasn't like this until "perpetual testing initiative" update. Pause menu had "Load Game" dialog.
